Resources/Locale/en-US/job/role-timers.ftl

```diff
@@ -0,0 +1,5 @@
+role-timer-department-insufficient = Require {$time} more minutes in {$department} department.
+role-timer-overall-insufficient = Require {$time} more minutes of playtime.
+role-timer-role-insufficient = Require {$time} more minutes with {$job} for this role.
+
+role-timer-locked = Locked
```

Comment:
This probably should say more than just "Locked" w/o requiring a tool-tip hover, new players might just think the role is locked on the server.

Also, for the tooltips I think "Require" should be "Requires", but I'm not 100% sure.